app/workers/process_notification_event_worker.rb

```diff
@@ -36,9 +36,14 @@ def perform(user_id, event_id, system_name)
   # @return [Account]
   def create_notifications(event)
     provider = Provider.find(event.provider_id)
-
     if provider.suspended_or_scheduled_for_deletion?
-      Rails.logger.info "[Notification] skipping notifications for event #{event.event_id} of #{provider.state} account #{event.provider_id}"
+      Rails.logger.info "[Notification] skipping notifications for event #{event.event_id} to #{provider.state} provider #{event.provider_id}"
+      return
+    end
+
+    buyer = Account.find_by(id: event.try(:account_id))
+    if buyer&.suspended_or_scheduled_for_deletion?
+      Rails.logger.info "[Notification] skipping notifications for event #{event.event_id} from #{buyer.state} buyer #{buyer.id}"
```

Comment:
I don' think we need these log messages and at most they need to be `debug`.

Reply:
Done: https://github.com/3scale/porta/pull/4079/commits/96e62232347e871aff82dcc94dd3009930a57bfd